src/azure-cli/azure/cli/command_modules/appconfig/_format.py

```diff
@@ -81,25 +81,34 @@ def _configstore_credential_format_group(item):
 
 
 def _keyvalue_entry_format_group(item):
+    content_type = _get_value(item, 'contentType')
+    content_type = content_type if content_type != ' ' else _get_value(item, 'content_type')
+
```

Comment:
nit: Can you add some comments here describing the format change made by CLI core? Otherwise, it looks odd why we deal with these two fields separately.